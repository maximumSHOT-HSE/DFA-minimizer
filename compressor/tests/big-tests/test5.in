12 84 7
1 9 0
1 8 1
1 8 2
1 8 3
1 10 4
1 11 5
1 3 6
2 3 0
2 4 1
2 5 2
2 3 3
2 8 4
2 12 5
2 12 6
3 1 0
3 1 1
3 4 2
3 12 3
3 10 4
3 9 5
3 12 6
4 12 0
4 4 1
4 4 2
4 5 3
4 2 4
4 12 5
4 11 6
5 1 0
5 4 1
5 10 2
5 9 3
5 5 4
5 4 5
5 8 6
6 7 0
6 5 1
6 5 2
6 11 3
6 9 4
6 4 5
6 11 6
7 8 0
7 10 1
7 10 2
7 7 3
7 4 4
7 5 5
7 12 6
8 8 0
8 5 1
8 6 2
8 7 3
8 7 4
8 3 5
8 5 6
9 6 0
9 9 1
9 4 2
9 8 3
9 7 4
9 4 5
9 8 6
10 4 0
10 9 1
10 5 2
10 1 3
10 12 4
10 5 5
10 9 6
11 4 0
11 1 1
11 1 2
11 3 3
11 6 4
11 3 5
11 4 6
12 7 0
12 8 1
12 10 2
12 1 3
12 11 4
12 12 5
12 6 6
3
12
6 4 2 6 4 11 12 6 6 1 10 5 
