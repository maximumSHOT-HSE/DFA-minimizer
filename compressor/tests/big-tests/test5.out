12 84 7
3 1 0
5 1 0
3 1 1
11 1 1
11 1 2
10 1 3
12 1 3
4 2 4
2 3 0
2 3 3
11 3 3
8 3 5
11 3 5
1 3 6
10 4 0
11 4 0
2 4 1
4 4 1
5 4 1
3 4 2
4 4 2
9 4 2
7 4 4
5 4 5
6 4 5
9 4 5
11 4 6
6 5 1
8 5 1
2 5 2
6 5 2
10 5 2
4 5 3
5 5 4
7 5 5
10 5 5
8 5 6
9 6 0
8 6 2
11 6 4
12 6 6
6 7 0
12 7 0
7 7 3
8 7 3
8 7 4
9 7 4
7 8 0
8 8 0
1 8 1
12 8 1
1 8 2
1 8 3
9 8 3
2 8 4
5 8 6
9 8 6
1 9 0
9 9 1
10 9 1
5 9 3
6 9 4
3 9 5
10 9 6
7 10 1
5 10 2
7 10 2
12 10 2
1 10 4
3 10 4
6 11 3
12 11 4
1 11 5
4 11 6
6 11 6
4 12 0
3 12 3
10 12 4
2 12 5
4 12 5
12 12 5
2 12 6
3 12 6
7 12 6
3
8
1 2 4 5 6 10 11 12 
